modules/set-listeners/index.js

```diff
@@ -0,0 +1,35 @@
+import { displayBooks } from '../render-books/index.js';
+import navToSection from './nav-to-section.js';
+
+export default function setListeners(gBookShelf) {
+  document.addEventListener('click', (e) => {
+    if (e.target.matches('.remove-book')) {
```

Comment:
- [ ] Please, consider using the arrow function syntax in the whole code. For example, here you can go on with 

```
const setListeners = (gBookShelf) => {
....
```
and in the end, you can export it by default, as:

`export default setListeners;`